src/docserv/bih.py

```diff
@@ -244,11 +244,17 @@ def read_conf_dir(self):
         if not self.config['targets'][target]['active'] == "yes":
             logger.debug("Target %s not active.", target)
             return False
-        self.stitch_tmp_file = os.path.join(tempfile.mkdtemp(
-            prefix="docserv_stitch_"), 'docserv_config_full.xml')
+
+        self.stitch_tmp_dir = os.path.join(tempfile.gettempdir(), 'docserv_stitch')
+        try:
+            os.mkdir(self.stitch_tmp_dir)
+        except FileExistsError:
+            pass
```

Comment:
Lines 249-252 can be replaced by:

```python
# Move the following line at the top:
from contextlib import suppress

with suppress(FileExistsError):
    os.mkdir(self.stitch_tmp_dir)
```

For more details, see the [contextlib.suppress doc](https://docs.python.org/3/library/contextlib.html#contextlib.suppress).